siliconcompiler/core.py

```diff
@@ -787,6 +787,15 @@ def get(self, *keypath, field='value', job=None, cfg=None):
 
         """
 
+        # Ensure that all keypath values are strings.
+        # Scripts may accidentally pass in [None] if a prior schema entry was unexpectedly empty.
+        for kp in keypath:
+            if not type(kp) == str:
+                self.error(\
```

Comment:
Keep in mind that if `['option', 'continue']` is set this won't be fatal, so we'll probably bubble to another exception inside `_search()`. Might want to return early after with `None`?

Reply:
Good point, I hadn't tested with `('option', 'continue')` set to `True`. Added an early `return None` for that case.